### PR TITLE
[JS-Packages] Connection: Update RequiredPlan login button style

### LINKED ISSUE #22278: Jetpack Backup: Log in text alignment issue on connect button
### Impacted plugin

Backup

### Steps to Reproduce

1- Launch a Jurassic ninja site with Jetpack Backup
2- Check the Backup Admin page of an unconnected Jetpack Backup

### A clear and concise description of what you expected to happen.

Aligned text

### What actually happened

'log in' button  is not aligned:
image

### Other information

Issue found and discussed in this PR:
https://github.com/Automattic/jetpack/pull/21646

### Operating System

_No response_

### OS Version

_No response_

### Browser

Chrome/Chromium

### Browser Version(s)

_No response_

### PR DESCRIPTION
**Pull request by jwebbdev**
Fixes #22278 

#### Changes proposed in this Pull Request:

With the updates to the action button [here](https://github.com/Automattic/jetpack/commit/8e1d504bf44ef00e8f66106db6958c6c5feaf3ff)
the overrides in the Connection UI stopped working.  This patch updates them match the specificity of the action button update.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Launch the Jetpack Backup plugin in a disconnected state and view the admin page to ensure the Login link is correctly aligned.

Before:
image

After:
Screen Shot 2022-01-10 at 12 22 01 PM

### CODE REVIEW

**Comment by nunyvega** [comment]
I tested it locally on Chrome and Safari ( OS ) and it looked as expected ✅

**Review by nunyvega — APPROVED** [review]

File: projects/js-packages/connection/components/connect-screen/required-plan/style.scss [hunk 1]
```diff
@@ -56,7 +56,7 @@
 			text-decoration: underline;
 
 			width: auto;
-			min-width: 0;
+			min-width: fit-content;
```
Comment (sergeymitr) [inline projects/js-packages/connection/components/connect-screen/required-plan/style.scss:59]: Hi @jwebbdev, thanks for working on this!

`fit-content` doesn't work in Firefox.
We could use `-moz-fit-content`, but that would require the whole new level of CSS trickery, so I wouldn't recommend that either.
Is there a way to solve this without using `fit-content`?

Screen Shot 2022-01-10 at 1 39 09 PM

**Review by sergeymitr — CHANGES_REQUESTED** [review]

File: projects/js-packages/connection/components/connect-screen/required-plan/style.scss (hunk 1 above)
Comment (jwebbdev) [reply projects/js-packages/connection/components/connect-screen/required-plan/style.scss:59]: I can switch it back to 0.  I updated that based on another suggestion but I believe 0 worked fine as well.

**Review by sergeymitr — APPROVED** [review]
Nicely done 👍 🚢

**Comment by jwebbdev** [comment]
Added to backup/branch-1.1.0 in db64897137530c728ee8c8b643d9bec5ebd0f3dd

**Pull request merged**